modules/mtbseq/mtbseq.nf

```diff
@@ -4,13 +4,13 @@ nextflow.enable.dsl = 2
 // - tar -xvf GATK_TAR_FILE
 // - gatk-register gatk_folder/gatk_jar
 
-params.resultsDir_mtbseq_per_sample = "${params.outdir}/mtbseq/samples"
-params.saveMode_mtbseq_per_sample = 'copy'
-params.shouldPublish_mtbseq_per_sample = true
+params.results_dir_mtbseq_per_sample = "${params.outdir}/mtbseq/samples"
+params.save_mode_mtbseq_per_sample = 'copy'
+params.should_publish_mtbseq_per_sample = true
 
```

Comment:
@Mxrcon , would it be possible to separate the MTBSEQ processes into 2 modules, this we we can continue using the same pattern of `params.results_dir` all across.

Reply:
Once that is updated, we also need to update the config/params accordingly.

Reply:
Yes! I agree on that, I'll implement that and update the cofigurations/parametes after and them ping you again here!